### PR TITLE
Ensure Facebook button has the same width as the others in footer

### PR DESCRIPTION
**Pull request by tisderek**

### CODE REVIEW

File: assets/stylesheets/neighborly-style/templates/_footer.sass [hunk 1]
```diff
@@ -44,6 +44,14 @@
   .social-links
     margin-bottom: 5px
 
+    .facebook
```
Comment (danmelton) [inline assets/stylesheets/neighborly-style/templates/_footer.sass]: @gustavoguichard could you look at and accept?

Comment (gustavoguichard) [inline assets/stylesheets/neighborly-style/templates/_footer.sass]: :ok_hand:

**Comment by gustavoguichard** [comment]
It seems to work.
Although, the SG has a very modular "button framework", you can build any kind of button composing classes.
If we ever change 1 or 2 lines in the general button classes, or even the paddings in the [settings](https://github.com/neighborly/style/blob/master/assets/stylesheets/neighborly-style/core/_settings.sass#L128), or if we change the sentence for some reason, this will probably brake. Because this is a fixed solution while every button is based on a lot of base paddings and `em`'s.

I also think this can be accomplished with much less CSS, maybe just increasing the `padding-right: $bt-padding-xs-horizontal + XXpx` and if the text goes to the right you can pull it back to the left with `text-ident: -XXpx;`.

What we want here is a bit hacky: corrupt the button width so it looks the same width as the others.
The other solutions that come to my mind are:
- Add couple `&nbps;` after `Like` (:star: hacky classy)
- Use the grid with `.btn-block` class: `<.row><.col-xs-6><a.btn.btn-block.twitter /></.col-xs-6><.col-xs-6><a.btn.btn-block.facebook />...` (:-1: verbose)
- Flexbox... (:surfer: feeling adventurous)

**Comment by danmelton** [comment]
@tisderek @gustavoguichard let's get this change in please

**Comment by gustavoguichard** [comment]
=/

**Comment by tisderek** [comment]
Hey guga I hadn't seen your long comment - just the :ok hand: and Dan's
approval. That's why I went ahead and merged it, I'm sorry for having
overlook your comment and made you go  =/

Let's connect asap about this issue, I'm sorry again for missing your
comments

On Fri, Dec 11, 2015 at 10:12 AM, Gustavo Guichard <notifications@github.com

**Pull request merged**